CENNZnet_format.md

```diff
@@ -145,7 +122,7 @@ The domain is binary encoded. It contains a version definition, and a permission
     3.  Named module constraints take priority over wildcard module constraints
 2.  Any contract named `"*"` is a wildcard
     1.  A wildcard contract enables all contracts
-    2.  Constraints applied to a wildcard contract such as `methods` and `block_cooldown` apply
+    2.  Constraints applied to a wildcard contract such as `block_cooldown` apply
```

Comment:
just realized the term 'constraints' is confusing. Should probably change that to say pact bytecode at some point